rules/prefer-array-index-of.js

```diff
@@ -75,12 +75,8 @@ const create = context => {
 
 			if (
 				parameter.type !== 'Identifier' ||
```

Comment:
parameter always `Identifier`, L22 and L31

Reply:
And `passiveExpression` always truly.

Reply:
OK, useless check.